core_ml_modules/language_processing/word.py

```diff
@@ -0,0 +1,32 @@
+class Word:
+    """
+    Word object containing its original form, canonical form (head of the cluster it belongs to), and part-of-speech tag.
+    :type original_form: str
+    :type canonical_form: str
+    :type pos_tag: str
+    """
+    original_form = None
+    canonical_form = None
+    pos_tag = None
+
+    def __init__(self, original_form):
+        self.original_form = original_form
+
+    def set_canonical_form(self, canonical_form):
```

Comment:
I think this class needs getters as well - otherwise there's no indication that I can't just reach in, use the properties of the object, including by changing them rather than using these methods